Add logspace test modules

tugsbayasgalan opened the pull request:
Differential Revision: D33885099

cccclai commented:
We probably should add more description about the pr.

cccclai commented on test/jit/test_upgraders.py:
```diff
@@ -131,6 +138,38 @@ def test_func():
+        for (a, b) in sample_inputs:
+            output_with_step, output_without_step = loaded_model(a, b)
+            # when no step is given, should have used 100
+            self.assertTrue(output_without_step.size(dim=0) == 100)
+            self.assertTrue(output_with_step.size(dim=0) == 5)
```
It's a bit tedious, but it's still better to use variables here, like `expected_output = 100`, self.assertTrue(actual_output, expected_output)`.

tugsbayasgalan committed:
Update linspace and bump version nuymber to 8 (#71486)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/71486

This PR adds upgraders for linspace and linspace.out as the optional step size will be deprecated soon. Old models will be using steps size of 100 when nothing is provided.

Test Plan: buck-out/gen/caffe2/test/jit#binary.par -r TestUpgraders.test_aten_linspace

Reviewed By: cccclai, mruberry

Differential Revision: D33654308

fbshipit-source-id: 056c3b2ded56c6f0b43dca55d1c55ea645fd74c1

tugsbayasgalan committed:
Add logspace test modules (#72052)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/72052

Test Plan: tofu_demands_tests

Reviewed By: cccclai

Differential Revision: D33885099

fbshipit-source-id: 4c905b8c32faa6756c40c2c4795082a3ce1c2c36
